docs/lib/predictions/fragments/android/escapehatch.md

```diff
@@ -0,0 +1,47 @@
+If you need functionality in the AWS services used by the Amplify Predictions category that isn't available, we provide an escape hatch so you can get a reference to that service. For example, to get a reference to `AmazonRekognitionClient`:
+
+```java
+// Obtain reference to the plugin
+final AWSPredictionsPlugin predictionsPlugin;
+try {
+    predictionsPlugin = (AWSPredictionsPlugin) Amplify.Predictions.getPlugin("awsPredictionsPlugin");
+} catch (ClassCastException error) {
+    Log.e("PredictionsQuickstart", "Unable to cast to AWSPredictionsPlugin");
+    return;
+}
+AWSPredictionsEscapeHatch escapeHatch = predictionsPlugin.getEscapeHatch();
+
+// Send a new request to the Rekognition endpoint directly using the client
+AmazonRekognitionClient client = escapeHatch.getRekognitionClient();
+CreateCollectionRequest request = new CreateCollectionRequest()
+        .withCollectionId("<new-collection-id-here>");
+client.createCollection(request);
+```
+
+In addition to `AmazonRekognitionClient`, this same pattern can be used to get access to `AmazonTranslateClient`, `AmazonPollyClient`, `AmazonComprehendClient`, and `AmazonTextractClient`. For example:
+
+```java
+// Obtain reference to the plugin
+final AWSPredictionsPlugin predictionsPlugin;
+try {
+    predictionsPlugin = (AWSPredictionsPlugin) Amplify.Predictions.getPlugin("awsPredictionsPlugin");
+} catch (ClassCastException error) {
+    Log.e("PredictionsQuickstart", "Unable to cast to AWSPredictionsPlugin");
+    return;
+}
+AWSPredictionsEscapeHatch escapeHatch = plugin.getEscapeHatch();
+
+// Obtain a reference to each of the supported Amazon service clients
+AmazonRekognitionClient rekognitionClient = escapeHatch.getRekognitionClient();
+AmazonTranslateClient translateClient = escapeHatch.getTranslateClient();
+AmazonPollyClient pollyClient = escapeHatch.getPollyClient();
+AmazonComprehendClient comprehendClient = escapeHatch.getComprehendClient();
+AmazonTextractClient textractClient = escapeHatch.getTextractClient();
+```
```

Comment:
Can we tie this to the below? With this client, you can then... some bridging sentence.